Tools/ML/model.h

```diff
@@ -20,23 +20,23 @@
 #ifndef TOOLS_ML_MODEL_H_
 #define TOOLS_ML_MODEL_H_
 
-// C++ and system includes
+#include <onnxruntime_c_api.h>
 #if __has_include(<onnxruntime/core/session/onnxruntime_cxx_api.h>)
 #include <onnxruntime/core/session/experimental_onnxruntime_cxx_api.h>
 #else
 #include <onnxruntime_cxx_api.h>
 #endif
-#include <vector>
-#include <string>
-#include <memory>
-#include <map>
-#include <algorithm>
 
-// ROOT includes
-#include "TSystem.h"
+#include <fairlogger/Logger.h>
```

Comment:
This one should be fixed in Framework/Logger.h

Reply:
Fixed